### DISCUSSION TITLE
New Schedule Page - List View Needs Sort Config Option

### DISCUSSION THREAD

**Discussion by LeeThompson** [pymedusa/Medusa#9519] [Ideas & Feature requests]
The new schedule page is interesting and I prefer "List" but now it defaults to descending date instead of ascending.  Oddly the other views (Poster, etc) do have a setting for sort order.

**Comment by p0psicles** [comment]
I'll make sure it will save your sort order. You can click the column header to change its sort order

**Reply by walterhpdx** [reply]
Saving the sort order is definitely needed.

But beyond that, the data is not correct.  First of all, the time always says 5pm, no matter what.  Second, for the three episodes here on the schedule today, it says they're for 5/12/21 - but today is 5/13/21.

![image](https://user-images.githubusercontent.com/6200380/118127354-6e3b6600-b3ae-11eb-9524-a93b25115471.png)

**Reply by p0psicles** [reply]
Can you run develop. As most of the issues should already be fixed.
